recipe/meta.yaml

```
@@ -143,10 +147,13 @@ about:
     This library is not meant to stand-alone. Instead it defines common helpers used by
     all Google API clients. For more information, see the documentation.
   doc_url: https://googleapis.dev/python/google-api-core/latest/index.html
-  doc_source_url: https://github.com/googleapis/python-api-core/tree/main/docs
   dev_url: https://github.com/googleapis/python-api-core
 
 extra:
+  skip-lints:
+    - outputs_not_unique
```

```suggestion
    - outputs_not_unique
```
This particular skip worries me since the outputs should be unique, right?

---

Addressed through refactor.